app/models/work_activity.rb

```diff
@@ -298,8 +298,9 @@ def body_html
       change_label = message_hash["change_label"]
       change_label ||= "Change"
       # TODO: Make this show the change label with the note under see changes
-      # See an example on line 171 of this file or on github: https://github.com/pulibrary/pdc_describe/blob/3578f734dcee092c15a9ff1fa778f20a846b47f3/app/models/work_activity.rb#L171
-      "#{change_label}: #{message}"
+      # See an example on line 173, message goes in mapped.join, of this file or on github: https://github.com/pulibrary/pdc_describe/blob/3578f734dcee092c15a9ff1fa778f20a846b47f3/app/models/work_activity.rb#L171
```

Comment:
I believe this should be removed
```suggestion
```

Reply:
Hi Carolyn, I have just made these changes. Thanks